suitcase/tiff_stack/__init__.py

```diff
@@ -61,12 +55,6 @@ def export(gen, directory, file_prefix='{uid}-', stack_images=True,
         descriptive value depends on the application and is therefore left to
         the user.
 
-    stack_images : Boolean, optional
-        This indicates if we want one image per file (`stack_images` = `False`)
-        or many images per file (`stack_images` = `True`). If using
-        `stack_images` = `False` then an additional image number is added to
-        the file name.
-
```

Comment:
Are we missing the `byteorder` and `imagej` documentation here?

Reply:
@mrakitin they appear to me to be just below in lines 61 and 64.

Reply:
Ah, I see now.